app/models/course/group.rb

```diff
@@ -27,8 +28,8 @@ def set_defaults
   # @return [Boolean]
   def should_create_manager?
     course && creator &&
-      course.course_users.exists?(user: creator) &&
-      !group_users.exists?(user: creator)
+      course.course_users.exists?(user_id: creator.id) &&
```

Comment:
Okay got it. Actually no change was required, will change back >_< Sorry. The `group_user.user` was also meant for the next line.